networkx/algorithms/euler.py

```diff
@@ -15,10 +21,15 @@
 
 from ..utils import arbitrary_element
 
+from collections import deque
+
 __author__ = """\n""".join(['Nima Mohammadi (nima.irt[AT]gmail.com)',
-                            'Aric Hagberg <hagberg@lanl.gov>'])
+                            'Aric Hagberg <hagberg@lanl.gov>',
```

Comment:
I don't see a problem with having both a copyright and an author.  But that is optional as far as I am concerned.